editor/editor_properties_array_dict.cpp

```diff
@@ -776,25 +776,26 @@ void EditorPropertyDictionary::_change_type(Object *p_button, int p_index) {
 }
 
 void EditorPropertyDictionary::_add_key_value() {
-	// Do not allow nil as valid key. I experienced errors with this
-	if (object->get_new_item_key().get_type() == Variant::NIL) {
-		return;
```

Comment:
Do you know what error the author was talking about? #20400 doesn't clearly explain what the error was.

Reply:
Not a clue; it's not mentioned anywhere in that PR's changes/discussion.